forge.config.ts

```diff
@@ -153,6 +154,23 @@ const config: ForgeConfig = {
 
 			return packageJson;
 		},
+		postPackage: async (_forgeConfig, options) => {
+			if (options.platform === 'linux') {
+				// symlink the etcher binary from balena-etcher to balenaEtcher to ensure compatibility with the wdio suite and the old name
+				await new Promise<void>((resolve, reject) => {
+					exec(
+						`ln -s "${options.outputPaths}/balena-etcher" "${options.outputPaths}/balenaEtcher"`,
+						(err) => {
+							if (err) {
+								reject(err);
+							} else {
+								resolve();
+							}
+						},
+					);
+				});
+			}
+		},
```

Comment:
I don't know about this one.

WDIO knows that we build with forge and find the build without problem for the two other platforms.
On linux as the name is different, it can't find it.

I thought of 4 potential solutions:
- set a conditional absolute path to the build in wdio.conf
- use a conditional env var in the action.yml
- create a symlink in the action.yml
- this one, create a symlink in forge post-package hook

This has my preference as it's the authoritative file for both the name of the package and the output path

WDYT?